03-Bash/kernel_git/kernel_stats.sh

```diff
@@ -26,6 +26,13 @@ count_files_in_dir(){
 	echo "count *.py :\t $(find "$1" -type f -name "*.py" | wc -l)"
 }
 
+count_comits(){
+	local tmp_dir="$(pwd)"
```

Comment:
You can use pushd/popd for same purposes.

Reply:
These commands pushd/popd did not work on my /bin/sh

Reply:
"pushd is a bash enhancement to the POSIX-specified Bourne Shell"